src/app/shared/store/provider.state.ts

```
@@ -945,4 +972,127 @@ export class ProviderState {
+  ): Observable<SubjectModel> {
+    patchState({ isLoading: true });
+    return this.studySubjectService
+      .getStudySubjectById(payload.subjectId, payload.providerId)
+      .pipe(tap((selectedSubject: SubjectModel) => patchState({ selectedSubject, isLoading: false })));
+  }
+
+  @Action(providerActions.UpdateStudySubject)
+  UpdateStudySubject(
+    { dispatch }: StateContext<ProviderStateModel>,
+    { payload }: providerActions.UpdateStudySubject
+  ): Observable<SubjectModel | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)